orderer/common/multichannel/registrar.go

```diff
@@ -1000,6 +1020,16 @@ func (r *Registrar) removeSystemChannel() error {
 	r.systemChannel.Halt()
 	delete(r.chains, systemChannelID)
 
+	// remove system channel resources
+	err := r.ledgerFactory.Remove(systemChannelID)
+	if err != nil {
+		return errors.WithMessagef(err, "error removing ledger for system channel %s", r.systemChannelID)
+	}
+
+	// remove system channel references
+	r.systemChannel = nil
+	r.systemChannelID = ""
+
```

Comment:
I suggest to move this 
`logger.Infof("Removed system channel: %s", systemChannelID)`
from the bottom to here.

Reply:
🧹